services/admin/promo/create_test.go

```diff
@@ -0,0 +1,67 @@
+package promo_test
+
+import (
+	"log"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/yitsushi/go-misskey/core"
+	"github.com/yitsushi/go-misskey/services/admin/promo"
+	"github.com/yitsushi/go-misskey/test"
+)
+
+func TestService_Create(t *testing.T) {
+	client := test.MakeMockClient(test.SimpleMockOptions{
+		Endpoint:     "/api/admin/promo/create",
+		RequestData:  &promo.CreateRequest{},
+		ResponseFile: "create.json",
+		StatusCode:   http.StatusOK,
+	})
+
+	err := client.Admin().Promo().Create(promo.CreateRequest{
+		NoteID:    "noteID",
+		ExpiresAt: time.Now().Add(86400 * 24 * time.Hour).Unix(),
+	})
+	if !assert.NoError(t, err) {
+		return
+	}
+}
+
+func TestService_Create_Error(t *testing.T) {
+	client := test.MakeMockClient(test.SimpleMockOptions{
+		Endpoint:     "/api/admin/promo/create",
+		RequestData:  &promo.CreateRequest{},
+		ResponseFile: "create.json",
+		StatusCode:   http.StatusOK,
+	})
+
+	// success
+	err := client.Admin().Promo().Create(promo.CreateRequest{
+		NoteID:    "noteID",
+		ExpiresAt: time.Now().Add(86400 * 24 * time.Hour).Unix(),
+	})
+	if !assert.NoError(t, err) {
+		log.Printf("[Admin/Promos/Create] %s", err)
+
+		return
+	}
+
+	// error
+
+}
+
+func TestPromoRequest_Validate(t *testing.T) {
+	test.ValidateRequests(
+		t,
+		[]core.BaseRequest{
+			promo.CreateRequest{},
+		},
+		[]core.BaseRequest{
+			promo.CreateRequest{NoteID: "8zwxx3cpy7", ExpiresAt: 0},
+			promo.CreateRequest{NoteID: "8zwxx3cpy8", ExpiresAt: 0},
+			promo.CreateRequest{NoteID: "8zwxx3cpy9", ExpiresAt: time.Now().Add(86400 * 24 * time.Hour).Unix()},
+		},
+	)
+}
```

Comment:
Can you please add an example in the test, like other [endpoints has](https://github.com/yitsushi/go-misskey/blob/74f1c29bd87879642f3a75d429506a2d5e65126f/services/promo/mark_as_read_test.go#L29)? That way we always have simple examples for all endpoints that can be copy pasted if someone wants to play around.

Reply:
Have added an example of creating a promo request, following the example.